controller/upload_file.rst

```diff
@@ -362,6 +363,10 @@ automatically upload the file when persisting the entity::
             if ($file instanceof UploadedFile) {
                 $fileName = $this->uploader->upload($file);
                 $entity->setBrochure($fileName);
+            } elseif($file instanceof File) {
```

Comment:
it should be : 
`elseif ($file instanceof File) {`

Reply:
Thank you @Simperfit updated!